c172p-set.xml

```diff
@@ -678,6 +679,17 @@ http://forum.flightgear.org/viewtopic.php?f=4&t=25157
             <oil-level type="double">7.0</oil-level>
             <oil-lacking type="double">0.0</oil-lacking>
             <oil_consumption_allowed type="bool">false</oil_consumption_allowed>
+			<oil-powergain-factor type="double">1</oil-powergain-factor>
```

Comment:
Here also many tabs. You should see in Github that the indentation is odd compared to the surrounding code.